src/resources/js/v2/tickets-block.js

```diff
@@ -607,6 +611,89 @@ tribe.tickets.block = {
 		);
 	};
 
+	/**
+	 * Toggle the ticket item description visibility.
+	 *
+	 * @since TBD
+	 *
+	 * @param {event} trigger The event.
+	 *
+	 * @return {void}
+	 */
+	obj.itemDescriptionToggle = function( trigger ) {
+		if ( ! trigger ) {
+			return;
+		}
+
+		const $trigger = $( trigger );
+
+		if (
+			! $trigger.hasClass( obj.selectors.itemDescriptionButtonMore.className() ) &&
+			! $trigger.hasClass( obj.selectors.itemDescriptionButtonLess.className() )
+		) {
+			return;
+		}
+
+		const $parent = $trigger.closest( obj.selectors.item );
+		const $target = $( '#' + $trigger.attr( 'aria-controls' ) );
+
+		if ( ! $target || ! $parent ) {
```

Comment:
This won't work as intended, an empty jQuery object will return `true`. Instead, use `$target.length`.